packages/amplication-client/src/VersionControl/BuildDeployments.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { Icon } from "@rmwc/icon";
+import * as models from "../models";
+
+import { SHOW_DEPLOYER } from "../feature-flags";
+import "./BuildDeployments.scss";
+
+const CLASS_NAME = "build-deployments";
+
+type Props = {
+  build: models.Build;
+};
+
+const BuildDeployments = ({ build }: Props) => {
+  if (!SHOW_DEPLOYER) return null;
```

Comment:
I think it’s better to be checked outside the component

Reply:
There is ofcource another check outside the component. 
I decided to add another check inside just to validate even if the dialog is used somewhere else in the future